src/python_testing/TC_IDM_1_4.py

```diff
@@ -74,7 +74,7 @@ async def test_TC_IDM_1_4(self):
         cap_for_batch_commands = 100
         number_of_commands_to_send = min(max_paths_per_invoke + 1, cap_for_batch_commands)
 
-        invalid_command_id = 0xffff_ffff
+        invalid_command_id = 0xfff4_00ff
```

Comment:
Please add a comment above saying that the prefix 0xfff4 is a reserved as "Test Vendor MC", and that from the perspective of a test device that is an invalid command id.

Reply:
Please also include in the comment that the spec is undefined with respect to decoding vs processing the invoke request so we use a decodable, so we used a valid id that should not be implemented on a production device as it is a Test Vendor MC for prefix 0xfff4

Reply:
added

Reply:
Thanks!